src/Illuminate/Container/BoundMethod.php

```diff
@@ -28,9 +28,7 @@ public static function call($container, $callback, array $parameters = [], $defa
         }
 
         return static::callBoundMethod($container, $callback, function () use ($container, $callback, $parameters) {
-            return call_user_func_array(
-                $callback, static::getMethodDependencies($container, $callback, $parameters)
-            );
+            return $callback(...static::getMethodDependencies($container, $callback, $parameters));
```

Comment:
This is a breaking change.

```
@livewire('my-component', ['foo' => 'bar'])
```

makes a call where the return value of the `getMethodDependencies()` method is an array with string keys.

Resulting in an exception:

> Cannot unpack array with string keys

Reply:
Can you please open an issue so we can focus the discussion?